src/main/java/com/guidetrack/mentorship_tracker/dao/requests/AdminSignupRequest.java

```diff
@@ -0,0 +1,4 @@
+package com.guidetrack.mentorship_tracker.dao.requests;
```

Comment:
these classes have nothing to do with data access. Did you mean dto for the package name?